.changeset/clean-otters-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+'@strapi/design-system': major
+---
+
+Enhancement: removed field wrapper components from inputs to streamline
```

Comment:
try use lowercase, the titles mess with commitlint, also for breaking changes you add `!` to the end, so this becomes:

```
enhancement!: remove field wrapper components from inputs
```